sdk/Lusid.Sdk.Tests/tutorials/Instruments/DemoInstrument.cs

```diff
@@ -0,0 +1,278 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    public class DemoInstrument: TestDataUtilities
```

Comment:
Agreed. I wanted to spearate the this as well. The way the apis are called is mixed throughout the tests - some call directly and others call through methods defined in TestDataUtilities. I could remove the inheritance of TestDataUtilities and just define a _testDataUtilities property on the DemoInstrumentClass that will create all the api endpoints. The other thought I had was to initialize all the api endpoints in the DemoInstrument class and then for any method call in TestDataUtilities I would pass in the required api... But that would mean re-writing all the other tests.

Reply:
Yes, we need to establish some kind of pattern here. I would probably do what you described in your last sentence. It allows them to make the choice of using the API directly, and it makes it very clear which API(s) a **_static_** utility function is using.

Reply:
We need to be mindful that these are meant to be **examples** rather than tests so the focus should be on clarity and ease of use for the reader in understanding how to structure and make the API calls. We should be adding in lots of comments to make it clear to reader what is happening and why where the code itself is not obvious. We write them as tests so we can run in pipelines and ensure they work

Reply:
Yes - they are examples first and tests second. As examples they need to be crystal clear and should be good reference for new users. I'll see how much work is required for the refactor decribed.